2006Redone Server/src/redone/game/players/Player.java

```diff
@@ -968,32 +968,9 @@ public void addToWalkingQueue(int x, int y) {
 		wQueueWritePtr = next;
 		// }
 	}
-
-	public boolean goodDistance(int objectX, int objectY, int playerX,
-			int playerY, int distance) {
-		for (int i = 0; i <= distance; i++) {
-			for (int j = 0; j <= distance; j++) {
-				if (objectId == 2282 || objectId == 10883 || objectId == 2322
-						|| objectId == 4493 || objectId == 12164
-						|| objectId == 1721 || objectId == 1722
-						|| objectId == 4304 && playerX == 2619
-						&& playerY == 3667) {
-					return true;
-				}
-				if (objectX + i == playerX
-						&& (objectY + j == playerY || objectY - j == playerY || objectY == playerY)) {
-					return true;
-				} else if (objectX - i == playerX
-						&& (objectY + j == playerY || objectY - j == playerY || objectY == playerY)) {
-					return true;
-				} else if (objectX == playerX
-						&& (objectY + j == playerY || objectY - j == playerY || objectY == playerY)) {
-					return true;
-				}
-			}
-		}
-		return false;
-	}
+public boolean goodDistance(int objectX, int objectY, int playerX, int playerY, int distance) {

```

Comment:
What's this?

Reply:
It's goodDistance method that I refactored. I don't even remember what it is.